packages/vite/src/node/plugins/html.ts

```diff
@@ -43,9 +45,13 @@ export const htmlProxyMap = new WeakMap<
   Map<string, Array<string>>
 >()
 
-export function htmlInlineScriptProxyPlugin(config: ResolvedConfig): Plugin {
+// HTML Proxy Transform result are stored by config
+// `${importer}_${query.index}` -> transformed css code
```

Comment:
We could probably add a comment for handling scripts as well.

Reply:
Ah I noticed this map is only for css, not for scripts. You can remove the extra comment I requested if you like.

Reply:
I added an example of this string to make it easier to understand.😀